docs/evaluate/development-features/versioning.mdx

```diff
@@ -0,0 +1,42 @@
+---
+id: versioning
+title: Versioning - Temporal Development Feature
+description: Learn how to implement versioning in your Temporal Workflows to manage changes in application code without causing non-deterministic behavior. Explore the benefits of versioning and how it ensures the continuity of long-running Workflows.
+sidebar_label: Versioning
+tags:
+  - versioning
+  - Workflow-versioning
+  - temporal-Workflows
+  - version-management
+keywords:
+  - temporal versioning
+  - manage Workflow changes
+  - version Workflow executions
+  - temporal versioning best practices
+  - Workflow continuity temporal
+  - version management
+  - temporal Workflow versioning
+---
+
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
+Versioning in Temporal is a feature that allows safely evolve your Temporal application code in production.
+Developers can make changes to their application code without causing non-deterministic behavior in current long-running Workflows. 
```

Comment:
We want to emphasize that this can only be done safely with versioning bc it sounds a bit like here that they're able to do that without

```suggestion
With versioning, developers are able to make changes to their application code without causing non-deterministic behavior in current long-running Workflows.
```